README.md

```diff
@@ -42,3 +43,28 @@ express.
 
 On a successful LTI launch the server returns a signed `accessToken` jwt that
 the client can then later use to authenticate saving content.
+
+# Management of Env Vars of the Deployments
+
+To use CLI to update any environment variable of the development, staging and
+production environments, follow these steps:
+
+1. Ask the admin to include you into the IONOS contract and update to policy of
+   the corresponding bucket. Alternatively, you can use the credentials of the
+   dev or admin user.
+2. Install a s3 client CLI (we recommend `s3cmd`) and configure it accordingly.
+   For access and secret keys go to
+   https://dcd.ionos.com/latest/#/key-management. Some other info:
```

Comment:
I cannot see the bucket when logging in using the `dev@serlo.org` account.

Reply:
try switching to "contract-owned buckets", then it should show.

Reply:
Ah nice! I did not see this small dropdown. Thank you!